Dockerfile

```diff
@@ -85,49 +73,41 @@ RUN apt-get -y install \
 # fonts-hanazono: chinese fonts
 # https://github.com/readthedocs/readthedocs.org/issues/6319
 RUN apt-get -y install \
-      fonts-symbola \
-      lmodern \
-      latex-cjk-chinese-arphic-bkai00mp \
-      latex-cjk-chinese-arphic-gbsn00lp \
-      latex-cjk-chinese-arphic-gkai00mp \
-      texlive-fonts-recommended \
-      fonts-noto-cjk-extra \
-      fonts-hanazono \
-      xindy
-
-# plantuml: is to support sphinxcontrib-plantuml
-# https://pypi.org/project/sphinxcontrib-plantuml/
-#
-# imagemagick: is to support sphinx.ext.imgconverter
-# http://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html
-#
-# rsvg-convert: is for SVG -> PDF conversion
-# using Sphinx extension sphinxcontrib.rsvgconverter, see
-# https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter
-#
-# swig: is required for different purposes
-# https://github.com/rtfd/readthedocs-docker-images/issues/15
-RUN apt-get -y install \
-      imagemagick \
-      librsvg2-bin \
-      plantuml \
-      swig
-
-# Install Python tools/libs
-ENV RTD_VIRTUALENV_VERSION 16.7.9
-RUN apt-get -y install \
-      python3-pip && \
-      pip3 install -U \
-      auxlib \
-      virtualenv==$RTD_VIRTUALENV_VERSION
+    fonts-symbola \
+    lmodern \
+    latex-cjk-chinese-arphic-bkai00mp \
+    latex-cjk-chinese-arphic-gbsn00lp \
+    latex-cjk-chinese-arphic-gkai00mp \
+    texlive-fonts-recommended \
+    fonts-noto-cjk-extra \
+    fonts-hanazono \
+    xindy
+
+# asdf Python extra requirements
+# https://github.com/pyenv/pyenv/wiki#suggested-build-environment
+RUN apt-get install -y \
+    liblzma-dev \
+    libncurses5-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libxmlsec1-dev \
+    llvm \
+    make \
+    python-openssl \
```

Comment:
Is `python-openssl` really required? This ends up installing Python 2.7 https://packages.ubuntu.com/focal/python-openssl

Reply:
Good point! It seems this is not required anymore. I doesn't appear in the wiki page linked.